packages/db/src/models/file.ts

```diff
@@ -22,6 +22,14 @@ class FilePgModel extends BasePgModel<PostgresFile, PostgresFileRecord> {
       .merge()
       .returning('*');
   }
+  searchByGranuleCumulusIds(
```

Comment:
a ready and efficient way wasn't already written to query files for multiple granules at once

Reply:
Docstring please

Reply:
Also: It's almost certainly worth improving this to allow field specification for efficiency reasons - no reason to return('*') if you only care about a specific field set.

Reply:
this is fixed to this effect.